pkg/devspace/kubectl/logs.go

```diff
@@ -82,11 +82,13 @@ func (client *client) LogMultipleTimeout(imageSelector []string, interrupt chan
 	for idx, pod := range pods {
 	Outer:
 		for _, container := range pod.Spec.Containers {
+			ImageSelectors:
 			for _, imageName := range imageSelector {
 				if imageName == container.Image {
 					reader, err := client.Logs(ctx, pod.Namespace, pod.Name, container.Name, false, tail, true)
 					if err != nil {
 						log.Warnf("Couldn't log %s/%s: %v", pod.Name, container.Name, err)
+						continue ImageSelectors
```

Comment:
Why use a named loop instead of just using continue since it is the most inner loop anyways?

Reply:
Honestly I don't think there's a right or wrong here, but I've changed it. 84b6491a110082e58e9800edc7649db1b9820dbd